docs/model-provider/index.mdx

```diff
@@ -0,0 +1,45 @@
+# Sourcegraph Model Provider
+
+<p className="subtitle">Learn how the default Sourcegraph LLM model provider enables AI features for Sourcegraph Enterprise customers.</p>
+
+The Sourcegraph model provider is the default and recommended way to configure AI features like [Deep Search](/deep-search) and [Cody](/cody). Through this service, we provide zero-configuration access to state-of-the-art models from various LLM providers, including [Anthropic](https://www.anthropic.com/) and [OpenAI](https://openai.com/), with enterprise-grade [privacy and security](#privacy-and-security).
+
+<Callout type="note">The Sourcegraph model provider is also referred to as "Cody Gateway", which is the LLM proxy service that provides this capability.</Callout>
+
+## Using the Sourcegraph model provider
+
+<Callout type="info">If you are a [Sourcegraph Cloud](/cloud/) customer, the Sourcegraph model provider is automatically configured by default. Other customers can verify their Enterprise subscription has access by confirming with their account manager.</Callout>
+
+To enable completions provided by the Sourcegraph model proivder on your Sourcegraph Enterprise instance, all you need to do is ensure your license key and your completions provider is set to `"sourcegraph"` is set in your [site configuration](/admin/config/site_config):
```

Comment:
```suggestion
To enable completions provided by the Sourcegraph model provider on your Sourcegraph Enterprise instance, all you need to do is ensure your license key and your completions provider is set to `"sourcegraph"` is set in your [site configuration](/admin/config/site_config):
```

Reply:
I think we should consider stepping away from the word completions .. I haven't heard it in the AI space for a long time. Inference?